package.json

```diff
@@ -43,13 +43,13 @@
     "react-native": ">= 0.30.0"
   },
   "devDependencies": {
-    "@types/lodash.debounce": "4.0.3",
+    "@types/lodash.debounce": "4.0.8",
     "@types/prop-types": "15.5.2",
-    "@types/react-native": "0.49.5",
     "@types/react": "16.4.7",
-    "@types/resize-observer-browser": "^0.1.7",
+    "@types/react-native": "0.49.5",
+    "@types/resize-observer-browser": "0.1.7",
     "file-directives": "1.4.6",
     "tslint": "5.11.0",
-    "typescript": "3.3.1"
```

Comment:
update typescript to a more recent version. because i couldnt get the tslint to run